src/plugin/regions.js

```diff
@@ -643,6 +643,7 @@ export default class RegionsPlugin {
         this.params = params;
         this.wavesurfer = ws;
         this.util = ws.util;
+        this.maxRegions = params.maxRegions;
```

Comment:
can you also document this at the top of this class, where the other params are documented?

Reply:
as I'm documenting this I realize the implementation I wrote only honors maxRegions when a user tries to create a region.  Wondering if it should be honored when a region is programmatically created....

Reply:
>  Wondering if it should be honored when a region is programmatically created....

I would assume it does..

Reply:
ok, maxRegions is now honored from the api too..